cli.go

```diff
@@ -290,6 +295,41 @@ func callRPCListTopicPeer(rpcAddr string, topics []string) {
 	fmt.Println(topicPeersString)
 }
 
+func callRPCListShardPeer(rpcAddr string, shards []string) {
+	conn, err := grpc.Dial(rpcAddr, grpc.WithInsecure())
+	if err != nil {
+		logger.Fatalf("Failed to connect to RPC server at %v, err: %v", rpcAddr, err)
+	}
+	defer conn.Close()
+	client := pbrpc.NewPocClient(conn)
+
+	topics := make([]string, len(shards))
+	for i := 0; i < len(shards); i++ {
+		topics[i] = "shardCollations_" + shards[i]
```

Comment:
@mhchia - I see that you originally define const collationTopicFmt = "shardCollations_%d" it in shardmanager.go. Do you want me to create a const.go file to add these magic string so I can avoid hardcoding a string here?

Reply:
I think it makes sense to have a `constants.go` to contain those constants. But for here, you can use `getCollationsTopic(shardID)` directly.

Reply:
Even better... thanks. I will do that instead